client/my-sites/site-settings/section-import.jsx

```diff
@@ -80,11 +73,7 @@ class SiteSettingsImport extends Component {
 	}
 
 	updateFromAPI = () => {
-		fetchState( this.props.site.ID );
-	};
-
-	updateState = () => {
-		this.setState( getImporterState() );
+		this.props.fetchState( this.props.site.ID );
```

Comment:
Can we create a `<QueryImporterState siteId />` component for this? That's how we trigger data fetching in components.

Also, guard for `this.props.site` being `null`. That's a pre-existing bug that can crash the `/settings/import/:site_id` page when the sites list is not yet loaded.